kmod/src/server.c

```diff
@@ -627,6 +702,14 @@ static int reclaim_log_trees(struct super_block *sb, u64 rid)
 					&lt.meta_avail) ?:
 	      alloc_move_empty(sb, &super->data_alloc, &lt.data_avail) ?:
 	      alloc_move_empty(sb, &super->data_alloc, &lt.data_freed);
+
+	if (lt.oino_bloom_ref.blkno && !ret) {
+		ret = scoutfs_free_meta(sb, &server->alloc, &server->wri,
+					le64_to_cpu(lt.oino_bloom_ref.blkno));
+		BUG_ON(ret);
+		lt.oino_bloom_ref.blkno = 0;
```

Comment:
This should clear the whole ref struct, not just the blkno.

Reply:
ok.. I'm gonna just set blkno and seq to 0 instead of using memset. lmk if you'd rather memset or some other struct initialize-to-0 syntax.